### ISSUE TITLE
Change PID #defines to variables

### ISSUE THREAD

**Issue by rwilbur** [inceptionev/FixMoreLungs#5]
We will need to change the PID #defines to variables so they can be changed via the communications to the raspberry pi

**Comment by lee-matthews** [comment]
Indeed. Was planning on making possible to set those too. Maybe create some kind of "engineering/debug" mode on the interface module so that we can play with these values directly that's protected from a typical user.

**Comment by rwilbur** [comment]
That makes sense. I am pretty sure that all these will need to be set in the UI but open to a user since nurses and doctors will need to be changing all of these based on the patient and their progress.

```
#define INSPIRE_TIME 1600
#define INSPIRE_RATE 1
#define PIP 142
#define INSPIRE_DWELL 800
#define INSPIRE_DWELL_PRESSURE 140
#define EXPIRE_TIME 1000
#define EXPIRE_RATE 1
#define PEEP 130
#define EXPIRE_DWELL 600
```
Perhaps with these we won't have these values being set directly but have it be a certain unit on the display and it gets converted to whatever these units are in the command.

**Comment by lee-matthews** [comment]
I was thinking of an engineering mode for the parameters for the PID for example, I don't think the medical personnel should have access to these. Same for any potential calibration parameters.
